### ISSUE TITLE
Exporting .glb file in a WebGL build in unity

### ISSUE THREAD

**Issue by PaulYoSwek** [atteneder/glTFast#625]
When trying to export my object that is placed inside a scene in Unity it never returns a success at all. 
The object I'm trying to export is just a simple cube.

**Note that it does work in editor but not in a build (WebGL build).**

The path looks like this: **/idbfs/e43848a1f09e340efa47c9c532e0b2ca/test.glb**

My code:

```
    public async void AdvancedExport()
    {
        var path = Application.persistentDataPath + "/test.glb";

        var exportSettings = new ExportSettings
        {
            Format = GltfFormat.Binary,
            FileConflictResolution = FileConflictResolution.Overwrite,
        };

        var export = new GameObjectExport(exportSettings);

        var rootLevelNodes = GameObject.FindGameObjectsWithTag("ExportMe");

        export.AddScene(rootLevelNodes, "My new glTF scene");

        var success = await export.SaveToFileAndDispose(path);

        Debug.Log("Success: " + success);
    }
```
The debug message success never gets called

**Comment by FernandoVillanuevaPadrones** [comment]
I am having the same issue. I think that it is because webgl does not support await/async calls. There is a package called UniTask that allows await to work but gltffast scripts cannot be modified so I do not what to do

**Comment by dkgupta2302** [comment]
Anyone found solution for webGL export with async/await? I am also facing same issue

**Comment by atteneder** [comment]
Fix is coming soon.

I looked at it and found out it hangs whenever `Stream.FlushAsync` is called in `GltfWriter`. Using `Flush` (non async) in WebGL only seems to fix it.

**Comment by atteneder** [comment]
Fix was released in [glTFast 6.8.0](https://docs.unity3d.com/Packages/com.unity.cloud.gltfast@6.8/changelog/CHANGELOG.html).
